### ISSUE TITLE
datalad extractor

### ISSUE THREAD

**Issue by remiadon** [bids-standard/BEP028_BIDSprov#39]
[a custom extractor](https://github.com/datalad/datalad-neuroimaging/pull/41) has been written for NIDM-results, should we do the same for BIDS-prov

**Comment by cmaumet** [comment]
Yes that's something we could investigate! In addition we could look into integrating BIDS-prov to record provenance when running `datalad run`.
